### PR TITLE
Command-line tool to help debug NumpyDocString

### PR DESCRIPTION
**Pull request by jnothman**

### CODE REVIEW

**Comment by stefanv** [comment]
Useful!

**Comment by jnothman** [comment]
It's like pydoc command-line tool but mangled...

**Comment by jnothman** [comment]
might make more sense as numpydoc.__main__

**Comment by jorisvandenbossche** [comment]
Works nicely!

**Comment by rgommers** [comment]
Nice!

PY2 is still unhappy.

**Comment by rgommers** [comment]
There's a problem here with submodules that are not imported by default into the main namespace. E.g.

    python -m numpydoc scipy.LowLevelCallable

works, but 

    python -m numpydoc scipy.signal.lfilter

gives `AttributeError: module 'scipy' has no attribute 'signal'`.

File: numpydoc/__main__.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    ap.add_argument('-c', '--config', type=_parse_config,
+                    action='append',
+                    help='key=val where val will be parsed by literal_eval, '
+                         'e.g. -c use_plots=True. Multiple -c can be used.')
+    args = ap.parse_args(argv)
+
+    parts = args.import_path.split('.')
+
+    for split_point in range(len(parts), 0, -1)[::-1]:
```
Comment (jnothman) [inline numpydoc/__main__.py]: Of course. This does the reversal twice. Silly me.

**Comment by jnothman** [comment]
Merging shouldn't hurt; we can iterate later if necessary.

**Comment by bsipocz** [comment]
This command line tool is brilliant, saved me lots of time by avoiding to build the full docs. Thank you.

**Pull request merged**